### DISCUSSION TITLE
when will the new  createSchema be released

### DISCUSSION THREAD

**Discussion by ramondebeijer** [samchungy/zod-openapi#334] [General]
hi @samchungy any idea when the new createSchema will be released? it is exactly what I am looking for

**Comment by samchungy** [comment]
Hey @ramondebeijer thanks for using the library!

I'll be looking to do a release sometime tomorrow. I like to sleep things over before releasing

https://github.com/samchungy/zod-openapi/pull/330#issuecomment-2395730768

**Reply by ramondebeijer** [reply]
tnx for the release

**Comment by samchungy** [comment]
Available from 3.1.1
